components/organisms/ProfileTabBody.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Button,
+  CalendarSVG,
+  CogSVG,
+  EthTransparentSVG,
+  HeartSVG,
+  Modal,
+  Skeleton,
+} from "@ensdomains/thorin";
+import { ProfileRecordItem } from "../02-molecules";
+import { DatabaseIcon } from "../01-atoms";
+import { formatDate, formatHexAddress } from "@/lib/utils/formats";
+import { CoinInfo, DomainData } from "@/lib/domain-page";
+import { useAccount, useEnsName } from "wagmi";
+import { useState } from "react";
+import { EditResolverModalContent } from "./EditResolverModalContent";
+import { useRouter } from "next/router";
+import { excludeKeys } from "@/pages/domains/[name]";
```

Comment:
it doesn't seem to have been exported [here](https://github.com/blockful-io/external-resolver-dapp/blob/a23432aebbc4e10ec69ac59b14b0b6fd0363cac0/pages/domains/%5Bname%5D/index.tsx#L67C9-L67C20)

Reply:
@pikonha that file is not updated for some reason